common/pulp_docker/common/transport.py

```diff
@@ -0,0 +1,117 @@
+class Version:
+    """
+    Represents a version of a Skopeo version file.
```

Comment:
can we re-phrase this to skopeo directory transport version?
this version file is the version of directory transport, not skopeo version.

maybe also worth changing the class name.

Reply:
I fixed the docstring, but left the class as "Version" since it is scoped by the module "transport".